doc/whats_new/v0.23.rst

```diff
@@ -85,6 +85,13 @@ Changelog
   :class:`linear_model.RidgeClassifierCV`.
   :pr:`15653` by :user:`Jérôme Dockès <jeromedockes>`.
 
+:mod:`sklearn.model_selection`
+..............................
+
+- |Enhancement| :func:`_fit_and_score` yields stack trace information in fit 
```

Comment:
Since the "Whats new" is user facing, I would prefer:

> :class:`model_selection.GridSearchCV` and :class:`model_selection.RandomizedSearchCV` now yields the stack trace information...

Reply:
Agreed